libs/metrics/top_level_metrics.py

```diff
@@ -13,11 +13,9 @@
 from api.can_api_v2_definition import TestPositivityRatioMethod, TestPositivityRatioDetails
 from libs import series_utils
 from libs.datasets.timeseries import OneRegionTimeseriesDataset
-from libs.metrics import icu_headroom
 from libs.metrics import icu_capacity
 
 Metrics = can_api_v2_definition.Metrics
-ICUHeadroomMetricDetails = can_api_v2_definition.ICUHeadroomMetricDetails
 # We will assume roughly 5 tracers are needed to trace a case within 48h.
 # The range we give here could be between 5-15 contact tracers per case.
 CONTACT_TRACERS_PER_CASE = 5
```

Comment:
NB: I'm guessing you can remove all the contact tracer code too in another PR if you're still feeling like cleaning up.